app/models/ngsi_subscription.rb

```diff
@@ -0,0 +1,9 @@
+class NgsiSubscription < ApplicationRecord
```

Comment:
I suggest keeping the same attrs and attrs names used in the [official definition](http://docs.orioncontextbroker.apiary.io/#reference/subscriptions). If not all attrs, at least the same names :)

Reply:
@taliaga Added the missing ones, though it's only one table to represent a subscription right now. Perhaps given subject and notification being their own specific JSON structure it could be useful to have them as their own tables rather than just stored as stringied JSONs. Then again, it's unlikely that a single sub will have so many entries in subject for instance to make it too large, so I could just keep it as is and let the UI generate the correct JSONs for subject and notification.

Reply:
Sure, I see no problem in having a single table for now. I was mostly concerned of having all the attrs in place and aligned with the Subscription model.

We'll need to be careful if required processing for a valid state is left behind for the UI, but we can see that later. 

Btw, is the "name" attr required by MasterMind or just a leftover?

Reply:
@taliaga It's for naming them when you make them in MasterMind, it's clearer for the user.